UnitsNet.Tests/CustomCode/AmountOfSubstanceTests.cs

```diff
@@ -42,6 +42,17 @@ public class AmountOfSubstanceTests : AmountOfSubstanceTestsBase
         protected override double NanopoundMolesInOneMole => 0.002204622621848776 * 1e9;
         protected override double PoundMolesInOneMole => 0.002204622621848776;
         protected override double MegamolesInOneMole => 1e-6;
+        
+        private static double MolarMassOfOxygen = 15.999;
+        private static double MolesInTenGramsOfOxygen = 0.6250390649415588;
```

Comment:
I see you re-define the same constants in different classes. I recommend moving these to a `Constants` class in its own file, where we can keep all the constants in one place to avoid having to duplicate them for test classes.